modules/core/src/main/java/org/apache/ignite/internal/processors/configuration/distributed/DistributedConfigurationProcessor.java

```diff
@@ -160,15 +216,12 @@ private static String toPropertyKey(String metaStorageKey) {
      * @return Public properties.
      */
     public List<DistributedChangeableProperty<Serializable>> properties() {
-        return props.values().stream()
-            .filter(p -> p instanceof DistributedChangeableProperty)
-            .map(p -> (DistributedChangeableProperty<Serializable>)p)
-            .collect(Collectors.toList());
+        return U.sealList(props.values());
```

Comment:
Do we really need to wrap in unmodifiable list here?
AFAIK we don't wrap for internal code.

Reply:
Currently, it's used only by control.sh to show list of properties, so overhead is very small. Also, `sealList` and `Collections.unmodifiable*` methods are used for internal code too, you can search for references of this method to ensure.